docs/dev/useful_models.rst

```diff
@@ -0,0 +1,17 @@
+Commonly Used Models
+====================
+Since [this picture](//web.mit.edu/benkraft/Public/esp-models.png) may not be the most useful way to figure out what model does what you want, here’s a list of the most commonly used ones:
+* ``esp.users.models.ESPUser``: This is a proxy model over ``django.contrib.auth.User`` (the standard user account model). Proxy model basically means it attaches extra methods, but not extra fields.
+* ``django.contrib.auth.models.Group``: Keeps track of types of users, like Student and Teacher.
+* ``esp.users.models.Permission``: Permissions for various parts of the website; can be applied to a particular ESPUser or Group.
+* ``esp.users.models.{StudentInfo,TeacherInfo,ContactInfo}``: Various types of extra information associated with various types of users.
+* ``esp.program.models.Program``: An instance of a program, like Splash 2014.
```

Comment:
For added clarity of what "instance" means, can you make that example be "Splash Fall 2014"?